### DISCUSSION TITLE
There‘s no package called ‚dplyr‘.

### DISCUSSION THREAD

**Discussion by 1E2V3A** [community/community#60157] [Discussions]
Does anybody have a solution for this problem? 
I use RStudio-2023.03.1-446 on a MacBooPro (2015), MacOS Monterey Version 12.6.6 and need this package for my data preparation, concretely to inner, left or outer join two tables with a different number of lines. However, the package do not exist on this Version of RStudio and a later Version RStudio-2023.06.0-421 I am not able to download.

**Comment by samus-aran** [comment]
Hi @1E2V3A ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [RStudio](https://github.com/rstudio/rstudio). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
